src/main/kotlin/no/nav/familie/klage/behandling/FerdigstillBehandlingService.kt

```diff
@@ -51,15 +54,31 @@ class FerdigstillBehandlingService(
             opprettJournalførBrevTask(behandlingId)
         }
         oppgaveTaskService.lagFerdigstillOppgaveForBehandlingTask(behandling.id)
-        behandlingService.oppdaterBehandlingsresultatOgVedtaksdato(behandlingId, behandlingsresultat)
+
+        val opprettetRevurdering = opprettRevurdering(behandlingId, behandlingsresultat)
```

Comment:
Vi skal vel strengt tatt bare gjøre dette hvis resultatet er `MEDHOLD`?

Reply:
Ja, hadde kanskje kunnet kalle den `opprettRevurderingForMedhold` for å gjøre det litt tydligere :)